cluster/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+rabbit1:
+  image: bijukunjummen/rabbitmq-server
+  hostname: rabbit1
+  labels:
+    io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=${stack_name}/rabbit2 
+  ports:
+    - 5672
+    - 15672
+
+rabbit2:
+  image: bijukunjummen/rabbitmq-server
+  hostname: rabbit2
+  labels:
+    io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=${stack_name}/rabbit1
+  links:
+    - rabbit1
+  environment:
+   - CLUSTERED=true
+   - CLUSTER_WITH=rabbit1
+   - RAM_NODE=true
+  ports:
+    - 5672
+    - 15672
+
+lb1:
```

Comment:
yay @davelnewton :). I think we're going to move this bit into a private repo though :)